websocket/_handshake.py

```diff
@@ -67,7 +67,7 @@ def _get_handshake_headers(resource, host, port, options):
     headers.append("GET %s HTTP/1.1" % resource)
     headers.append("Upgrade: websocket")
     headers.append("Connection: Upgrade")
-    if port == 80:
+    if port == 80 or ("host_only" in options and options["host_only"] == True):
```

Comment:
This is not my purpose. I would like to give user a way to disable adding 443 behind "host".